### ISSUE TITLE
[style]: social media icons should be positioned properly inside the profile card

### ISSUE THREAD

**Issue by harmeetsingh11** [priyankarpal/ProjectsHut#867]
### What's the style idea?

The social media icons should be positioned properly inside the profile card on the user's Projects Page

### Add screenshots

![f](https://github.com/priyankarpal/ProjectsHut/assets/130365071/ca153bf9-a825-47a1-baab-18eb70764839)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by priyankarpal** [comment]
but I'm not getting this problem 😢 

![image](https://github.com/priyankarpal/ProjectsHut/assets/88102392/00f2cf83-e855-4882-9ef3-a6af7664d9eb)

**Comment by priyankarpal** [comment]
yeah finally found this. do you wan to work on this issue? @harmeetsingh11 


![image](https://github.com/priyankarpal/ProjectsHut/assets/88102392/879a126a-4e29-456e-98e4-decc0d885bf3)

**Comment by harmeetsingh11** [comment]
Yes.

**Comment by priyankarpal** [comment]
> Yes.

assigned to you @harmeetsingh11

**Comment by priyankarpal** [comment]
Completed. Thanks for your contributions 🎉 ❤️
